README.md

```diff
@@ -121,21 +201,40 @@ arguments is invoked. This provides developers with sane defaults that are
 exactly identical to what is used to build the package itself. Also provided
 are mixins to compile with clang or ccache.
 
-### overrides
-
-Some packages may need extra overrides to ensure they build correctly, or
-changes to the source code may be necessary to accomodate their use in Nix.
-
-[nixros]: https://github.com/clearpathrobotics/nix-ros/
-[nix-ros-cachix]: https://app.cachix.org/cache/nix-ros#pull
-[gc]: https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-store-gc.html
-
 ## Disclaimer
 
 We're delighted to collaborate long-term with other users of Nix and ROS, however
 there's no commitment to support of the nix-ros and nix-ros-base repositories;
 these were prepared specifically for a talk given at ROSCon 2022.
 
+[nixros]: https://github.com/clearpathrobotics/nix-ros/
+[gc]: https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-store-gc.html
+
+
 ## Credits
 
-This work would not have been possible without @lopsided98 's [nix-ros-overlay](https://github.com/lopsided98/nix-ros-overlay/) repository.
+This work would not have been possible without @lopsided98's work on packaging
```

Comment:
I think we should just link to @lopsided98's github profile here, usernames don't become linkable in the readme with `@`.

Reply:
Yes good call, will fix.

Reply:
And for @lopsided98, no rush, but when you have time, we'd be delighted for you to look over this comparison to ensure it is fair and accurate. We're more than happy to accept corrections.

Reply:
It looks fine to me.